packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -116,80 +116,56 @@ def test_octarine_srv(cluster, timeout=30):
     cluster.deploy_marathon_app(app_definition)
 
 
-# By default telemetry-net sends the metrics about once a minute
-# Therefore, we wait up till 2 minutes and a bit before we give up
-def test_if_minuteman_routes_to_vip(cluster, timeout=125):
+def ensure_routable(cmd, service_points, timeout=125):
+    @retrying.retry(wait_fixed=2000,
+                    stop_max_delay=timeout*1000,
+                    retry_on_result=lambda ret: ret is False,
+                    retry_on_exception=lambda x: True)
+    def _ensure_routable():
+        proxy_uri = 'http://{}:{}/run_cmd'.format(service_points[0].host, service_points[0].port)
+        logging.info('Sending {} data: {}'.format(proxy_uri, cmd))
+        r = requests.post(proxy_uri, data=cmd)
+        logging.info('Requests Response: %s', repr(r.json()))
+        assert(r.json()['status'] == 0)
+    return _ensure_routable
+
+
+def test_if_minuteman_routes_to_vip(registry_cluster):
```

Comment:
registry_cluster should now be just cluster. registry_cluster is completely gone